### PR TITLE
Mandla sprint 1 user score

### LINKED ISSUE #5: as a user i can see my game statistics
Acceptance Criteria:

- [ ] The user is able to see game progress whenever requested.
- [ ] The user can see performance stats for all game tries.

### LINKED ISSUE #28: As a user i can see the game timer so that i know how much  time i am left with until the next game
Acceptance Criteria:

- [ ] The user can see how much time left until the next game.
- [ ] A user cannot see time if game is not yet played.

### PR DESCRIPTION
**Pull request by Khumalom966**
This is for score display user stories and the timer display 
Resolves #28 , resolves #5

### CODE REVIEW

**Review by Mulowwits — CHANGES_REQUESTED** [review]
- in scoreRoutes.js review your variables naming (I suggest you change student to player)

**Comment by Mulowwits** [comment]
> * more tests can be provided for next sprint
> * standard from code style guide has been followed (good)
> * in scoreRoutes.js review your variables naming (I suggest you change student to player)

**Comment by Khumalom966** [comment]
> * in scoreRoutes.js review your variables naming (I suggest you change student to player)

resolved

File: routes/scoreRoutes.js [hunk 1]
```diff
@@ -17,10 +17,5 @@ router.get('/api/get/:id', function (req, res) {
...
-// routes for POST requests
-router.post('/api/create', function (req, res) {
-  console.log('Adding the following student:', req.body.student)
-  classList.add(req.body.student)
-  res.redirect(req.baseUrl)
-})
-module.exports = router
\ No newline at end of file
+
+module.exports = router
```
Comment (Mulowwits) [inline routes/scoreRoutes.js:21]: Good to go

**Review by Mulowwits — APPROVED** [review]

**Pull request merged**